hub/powertoys/fancyzones.md

```diff
@@ -129,6 +133,9 @@ Canvas layout also has keyboard support for zone editing. Use the <kbd>arrow</kb
 
 ### Quickly changing between custom layouts
 
+> [!WARNING]
```

Comment:
There is no danger at all, please use "Note"

Reply:
agree but this could be a visual thing